src/Apps/Order/Routes/Shipping2/FulfillmentDetails.tsx

```diff
@@ -93,64 +96,52 @@ export interface ShipValues {
 export type FulfillmentValues = ShipValues | PickupValues
 
 export interface FulfillmentDetailsFormProps {
-  active: boolean
   onSubmit: (
     values: FulfillmentValues,
-    formikHelpers: FormikHelpers<FulfillmentValues>
+    formikHelpers?: FormikHelpers<FulfillmentValues>
   ) => void | Promise<any>
-  setFulfillmentFormHelpers: React.Dispatch<
-    React.SetStateAction<
-      Pick<
-        FormikProps<FulfillmentValues>,
-        "handleSubmit" | "isValid" | "values"
-      >
-    >
-  >
-  initialFulfillmentValues: FulfillmentValues
+
   me: FulfillmentDetailsForm_me$data
   order: FulfillmentDetailsForm_order$data
-  renderMissingShippingQuotesError: boolean
 }
 
 type AddressFormMode = "saved_addresses" | "new_address" | "pickup"
 
-export const FulfillmentDetails: FC<FulfillmentDetailsFormProps> = ({
-  active,
-  initialFulfillmentValues,
-  ...props
-}) => {
-  // // Current we do this from index (to refresh shipping quotes)
-  // const {
-  //   fulfillmentType: savedFulfillmentType,
-  //   fulfillmentDetails: savedFulfillmentDetails,
-  // } = useShippingContext()
-  // useEffect(() => {
-  //   if (savedFulfillmentType === FulfillmentType.SHIP) {
-  //     props.onSubmit(savedFulfillmentDetails)
-  //   }
-  // // eslint-disable-next-line react-hooks/exhaustive-deps
-  // }, [])
+export const FulfillmentDetails: FC<FulfillmentDetailsFormProps> = props => {
+  const { initialValues } = useShippingContext()
```

Comment:
blocking: Here and elsewhere -- now that even more data is being stored in the context -- lets update all context-based call-sites to read:

```tsx
const shippingContext = useShippingContext()
...
foo(shippingContext.initialValues)
```

Reply:
alright, will do